src/main/groovy/projet/gl51/store/MemoryProduct.groovy

```diff
@@ -0,0 +1,37 @@
+package projet.gl51.store
+
+class MemoryProductStorage implements ProductStorage {
+	
+	private List<Product> products = []
+
```

Comment:
Should be initialized with a constructor

Reply:
no added value and not necessary in groovy, groovy provides you a default one which does the work in 90% of the cases